packages/compile-solidity/src/compilerSupplier/loadingStrategies/VersionRange.ts

```diff
@@ -6,7 +6,19 @@ import originalRequire from "original-require";
 
 // must polyfill AbortController to use axios >=0.20.0, <=0.27.2 on node <= v14.x
 import "../../polyfill";
-import { default as axios, AxiosResponse } from "axios";
+import { default as axios_plain, AxiosResponse } from "axios";
```

Comment:
```suggestion
import { default as axiosPlain, AxiosResponse } from "axios";
```

We prefer camelcase variable names instead of kebabcase.